generator/osm2meta.cpp

```diff
@@ -287,6 +287,21 @@ string MetadataTagProcessorImpl::ValidateAndFormat_wikipedia(string v) const
   return normalized;
 }
 
+
+string MetadataTagProcessorImpl::ValidateAndFormat_wikimedia_commons(string v) const
+{
+    if(strings::StartsWith(v, "File:") or strings::StartsWith(v, "Category:"))
```

Comment:
The code is not formatted according to our style.

Reply:
Replaced or with ||.  Anything else?